packages/providers/test/AuthenticationProviderHttp.spec.ts

```diff
@@ -0,0 +1,79 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import { AuthenticationProviderHttp } from '../src';
+
+describe('AuthenticationProviderHttp', () => {
+  jest.useFakeTimers();
+
+  const CLIENT_ID = 'CLIENT_ID';
+  const CLIENT_SECRET = 'CLIENT_SECRET';
+  const ACCESS_TOKEN = 'ACCESS_TOKEN';
+  const EXPIRES_IN = 3600;
+  const AUDIENCE = 'https://audience.test';
```

Comment:
Wouldn't be nicer for the audience being the origin host name? I did this in Family but with many for each "secret".

Reply:
:man_shrugging: